src/components/widgets/Donation.astro

```diff
@@ -0,0 +1,103 @@
+---
+import WidgetWrapper from '~/components/ui/WidgetWrapper.astro';
+import AnimatedCrownSVG from '~/components/widgets/AnimatedCrownSVG.astro';
+import ParallaxSection from '~/components/widgets/ParallaxSection.astro';
+import type { Donation as Props } from '~/types';
+
+import donationBgImage from '~/assets/images/donation-bg.jpg';
+
+const {
+  title = {
+    firstPart: '',
+    secondPart: '',
+  },
+  subtitle = '',
+  donationText = '',
+  donationButtonText = '',
+  donationLink = '/',
+  features = [],
+  imageSrc = donationBgImage,
+  id,
+  isDark = false,
+  classes = {},
+  bg = await Astro.slots.render('bg'),
+} = Astro.props;
+
+const rotationClasses = [
+  'rotate-0', // برای index 0
+  'rotate-90', // برای index 1
+  '-rotate-90', // برای index 2
+  'rotate-180', // برای index 3
+];
+---
+
+<WidgetWrapper
```

Comment:
Could you please explain what does this component? WidgetWrapper?

Reply:
The GitHub link of the tutorial I made with the help of AI 
[https://github.com/alirezakariminejad/astrowind-WidgetWrapper-docs](url)